laravel/the-basics/databases.html.md.erb

```diff
@@ -515,8 +510,9 @@ To test whether your Laravel Fly App can successfully connect with your Upstash
 <li>Using `REDIS_URL` should override `REDIS_HOST`, add here the `REDIS_URL` received during `fly redis create`</li>
 <li>Upstash only supports "database 0", so update your `REDIS_CACHE_DB` config to 0. Otherwise, you will receive an error : "SELECT` failed: ERR Only 0th database is supported! Selected DB: 1..."</li>
 </ul>
-
-2) Then deploy your `fly.toml` changes:
+</li>
```

Comment:
Awesome work @jeregrine ! One thing here, is it possible to add a space separator between the line 513 and line 514, maybe with a <br> tag? Otherwise there is no space in between and it looks squeezed like so: 
__________________________________________________________

![image](https://user-images.githubusercontent.com/33826441/211870762-3d6ad531-1f46-46f4-88e7-72326365ae94.png)